README.md

```diff
@@ -53,6 +53,10 @@ As an example:
 If you want to keep your decentralized, shareable git remote in sync with your GitHub repo adding
 a simple github action as illustrated in [dgit-github-action](https://github.com/quorumcontrol/dgit-github-action) is all it takes.  Once completed your decentragit decentralized shareable remote will always be up to date and ready when you need it.<br>
 
+#### Publish to dg-pages
+New Feature! DGit now allows you to publish your frontend files (html, css, js, .vue, .react) files to Skynet.
+Just checkout your files into a new branch named dg-pages, commit and push!
+
```

Comment:
given the githook uses a the `skynet` executable, it should be noted in this section as a requirement.